content/reference/05-plugins/01-grafana.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Grafana Datasource Plugin"
+metaTitle: "Reference | Grafana Datasource Plugin "
+metaDescription: "Pixie's Grafana Datasource Plugin"
+order: 5
+---
+
+The Pixie Datasource plugin allows you to visualize data from the Pixie observability platform in Grafana.
+
+## Installation
+
+Instructions for installing the plugin can be found [here](https://github.com/pixie-labs/grafana-plugin/blob/main/README.md).
+
+## Usage
+
+To get started using the plugin, check out the [tutorial](/tutorials/grafana).
+
+### Configuration
+
+The plugin requires a Pixie API key and cluster ID to execute queries. Grafana encrypts these values.
+
+- To create an API key, follow the directions [here](/using-pixie/api-quick-start/#get-an-api-token).
+- To find your cluster's ID, follow the directions [here](/using-pixie/api-quick-start/#get-a-cluster-id).
+
+### Queries
+
+The plugin uses the Pixie Language (PxL) to query telemetry data collected by the Pixie platform.
+
+- Learn more about the PxL language [here](/reference/pxl/).
+- Learn how to write a PxL script [here](/tutorials/pxl-scripts).
+
+<Alert variant="outlined" severity="warning">
+  Note that the plugin does not support Vis Spec input. PxL queries must return a table using the `px.display()` call.
+</Alert>
+
+If there are syntax errors in your PxL query, Grafana will display an error message.
+
+If a query output table has a `time_` column, the plugin will automatically sort the column.
```

Comment:
Actually, if the query output table has a column with a timestamp in general, the plugin will sort the data by time.